Add features to UG #98 (#99)
AY2425S2-CS2103T-F10-1/tp: docs/UserGuide.md

## Quick start

1. Ensure you have Java `17` or above installed in your Computer.<br>

>    **Mac users:** Ensure you have the precise JDK version prescribed [here](https://github.com/AY2425S2-CS2103T-F10-1/tp/releases).

Reviewer: ```suggestion
   **Mac users:** Ensure you have the precise JDK version prescribed [here](https://se-education.org/guides/tutorials/javaInstallationMac.html).
```
I think this might be the wrong link. This link should be to install the correct JDK version while the next link is for the release.

> 1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).

Reviewer: ```suggestion
1. Download the latest `.jar` file from [here](https://github.com/AY2425S2-CS2103T-F10-1/tp/releases).
```

1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
   Some example commands you can try:

   * `list` : Lists all students.

>    * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a student named `John Doe` to TeamSync.

Reviewer: ```suggestion
   * `add -n John Doe -p 98765432 -e johnd@example.com -a John street, block 123, #01-01` : Adds a student named `John Doe` to TeamSync.
```
We should standardise the command format for this and other commands to be the same as our current release's.

Format: `help`

### Adding a student: `add`

> Adds a student to the address book.

Reviewer: ```suggestion
Adds a student to TeamSync.
```
It might be better to standardise this and other mentions of address book to TeamSync if appropriate.

* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`

### Listing all students : `list`

Shows a list of all students in the address book.

Format: `list`

### Editing a students : `edit`

Edits an existing students in the address book.

Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`

> * Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​

Reviewer: ```suggestion
* Edits the student at the specified `INDEX`. The index refers to the index number shown in the displayed student list. The index **must be a positive integer** 1, 2, 3, …​
```
It might be better to standardise this and other mentions of person to student if appropriate.

* `list` followed by `delete 2` deletes the 2nd person in the address book.
* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.

### Adding a module to a student: `addmod`

Adds a module to the specified student

Format: `addmod INDEX MODULE_CODE DAY START_TIME END_TIME`

* The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
* The module code will follow NUS module code format (Department tag + 4-digit number + optional last letter)

> * Module code is case-insensitive. e.g cs2103t and CS2103T will be regarded as the same module

Reviewer: ```suggestion
* Module code is case-insensitive.<br>
  e.g. cs2103t and CS2103T will be regarded as the same module
```
It might be better to add a dot after e.g

* `delmod 1 cs2101` deletes the module CS2101 from the 1st person in the address book.
  ![result for 'delmod 1 cs2101'](images/delmodCS2101Result.png)

### Add a meeting for the team: `meeting`

Creates a meeting with the specified date, start and end time

Format: `meeting DATE START_TIME END_TIME`

* DATE is in DD-MM-YYYY format
* Start time and End time is given in 24-hour format and in HH:MM

Examples:
* `meeting 27-03-2025 12:00 15:00` creates a new meeting on 27th March 2025 from 12pm to 3pm.
  ![result for 'meeting 27-03-2025 12:00 15:00'](images/addMeetingResult.png)

### Listing all meetings : `listmeetings`

Shows a list of all meetings in the address book.

Format: `listmeetings`

Examples:
* `listmeetings` lists all existing meetings in the address book.
  ![result for 'listmeetings'](images/listMeetingResult.png)

### Delete an existing meeting: `delmeeting`

Deletes the meeting identified by the index number used in the displayed meeting list

Format: `delmeeting INDEX`

* Deletes the meeting at the specified `INDEX`.
* The index refers to the index number shown in the displayed meeting list.
* The index **must be a positive integer** 1, 2, 3, …​

Examples:

> * `listmeeting` followed by `delete 1` deletes the 1st meeting in the address book.

Reviewer: ```suggestion
* `listmeetings` followed by `delete 1` deletes the 1st meeting in the address book.
```
The command should be `listmeetings`
